src/test/java/io/appium/java_client/ios/XCUIAutomationTest.java

```diff
@@ -91,4 +92,14 @@
         driver.rotate(landscapeLeftRotation);
         assertEquals(driver.rotation(), landscapeLeftRotation);
     }
+
+    @Test public void testTouchId() {
+        try {
+            driver.touchId(true);
+            driver.touchId(false);
+            assertEquals(true, true);
```

Comment:
Actual Usecase of API ```touchID``` is to unlock or sign in to any application using touchID. Since we dont have our test app built with any registration or sign in feature, we cannot use that scenario.

Otherwise Can we lock the simulator and unlock the simulator using ```driver.touchID(true)```.Ideal case this should be working fine.